content/blog/2024-08-23-roadmap-may-2024-retro.md

```diff
@@ -0,0 +1,245 @@
+
+### Progress beyond roadmap {#masonry-extras}
+
+Masonry is in a state of flux, however is starting to see a few exploratory users.
+To unblock these users, some additional important features were added outside of the roadmap, such as:
+
+- Support for embedding Masonry inside a pre-existing event loop ([xilem#417](https://github.com/linebender/xilem/pull/417)).
+- Support for a `gap` property in the flexbox ([xilem#437](https://github.com/linebender/xilem/pull/437)).
+- Fixes on iOS, including [xilem#418](https://github.com/linebender/xilem/pull/418) and [xilem#421](https://github.com/linebender/xilem/pull/421).
+- [xilem#233](https://github.com/linebender/xilem/pull/233) allowed our tests to be run on CI in a consistent manner.
+
+## Parley [(original)](@/blog/2024-06-17-roadmap-may-2024.md#parley)
+
+### Create abstractions for text selection and editing [(original)](@/blog/2024-06-17-roadmap-may-2024.md#write-pass-order-rfc#create-abstractions-for-text-selection-and-editing) ?
+
+- **Priority**: Highest
+- **Status**: There have been useful discussions in [parley#52](https://github.com/linebender/parley/issues/52).
+  Work has started on this in [parley#106](https://github.com/linebender/parley/pull/106) in response to this retrospective.
+
+### Inline block layout [(original)](@/blog/2024-06-17-roadmap-may-2024.md#inline-block-layout)
+
+- **Priority**: High
+- **Status**: A first cut of this feature was added by Nico Burns in ([parley#67](https://github.com/linebender/parley/pull/67)).
+  [parley#76](https://github.com/linebender/parley/pull/76) expands upon this, and has been approved in principle, but is awaiting thorough review.
+
+### Write adequate documentation [(original)](@/blog/2024-06-17-roadmap-may-2024.md#write-adequate-documentation)
+
+- **Priority**: High
+- **Status**: Some work was done towards this in [parley#26](https://github.com/linebender/parley/pull/26), which unfortunately could not be merged.
+  [parley#97](https://github.com/linebender/parley/pull/97) added some documentation to Fontique.
+  Further documentation work from Nico is currently blocked on the tree-like styles PR [parley#76](https://github.com/linebender/parley/pull/76) being merged (to avoid conflicts).
+
+### Software renderer [(original)](@/blog/2024-06-17-roadmap-may-2024.md#software-renderer)
+
+- **Priority**: Low
+- **Status**: This was not completed in the form envisioned.
+  However, Nico contributed two examples, showing how to use Parley with Swash ([parley#54](https://github.com/linebender/parley/pull/54)) and tiny-skia ([parley#55](https://github.com/linebender/parley/pull/55)).
+  We have decided not to carry this forward into the next roadmap.
+
+### Progress beyond roadmap {#parley-extras}
+
+[parley#56](https://github.com/linebender/parley/pull/56) added support for Emoji, which is used with the [Vello PRs](#vello-extras).
+
+<!-- TODO: Accurate sentence on Harfruzz -->
```

Comment:
Here's my proposed sentence:

Recently the [harfruzz] project, a fork of rustybuzz to be integrated fully with fontations, has emerged as an extremely promising direction for text shaping for the Rust ecosystem. We plan to use it as the basis for text layout.